Automattic/wp-calypso
# Jetpack Cloud: Add route to temporarily switch request type

#### Changes proposed in this Pull Request

* Add a route that forces Jetpack Cloud to temporarily switch from oauth to proxy-style requests

#### Testing instructions

1. Boot local Jetpack.com
2. Navigate to WordPress.com in a private tab and login
3. in the same tab navigate to `http://jetpack.cloud.localhost:3000/oauth-override`
4. verify you are taken to the `/backup` or `/scan` page for the primary site for the account
